content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -85,9 +85,9 @@ Because ClusterRoles are cluster-scoped, you can also use them to grant access t
 
 * cluster-scoped resources (like {{< glossary_tooltip text="nodes" term_id="node" >}})
 * non-resource endpoints (like `/healthz`)
-* namespaced resources (like Pods), across all namespaces
+* namespaced resources (like Pods), across all namespaces <br>
```

Comment:
Is there an example somewhere else in the Kubernetes documentation that handles this with a newline? I'm not sure what code you're suggesting to use. I searched for markdown guidance and see there are generally two methods; one, use the line break and two, use two trailing whitespaces. I've not seen the whitespaces used to add a line under a bullet and want to ensure this doesn't add the extra line spacing between the lines themselves since that would render as a bulleted item missing a bullet.

Reply:
Hey, @edwardrosen! Of course, one example that I can think of is this: https://github.com/kubernetes/website/blame/master/content/en/docs/reference/access-authn-authz/abac.md#L50

Where there are various nested bullets outlined, then newline/returns are used to separate before moving to the next section. 

Does that answer your question, or is there anything else I can clarify?

Reply:
you may want to avoid the br tag by inserting an empty line before line 39. In markdown, an empty line signals a new paragraph. It works even for list items.